paasta_tools/utils.py

```diff
@@ -2785,6 +2785,17 @@ def decompose_job_id(job_id: str, spacer: str = SPACER) -> Tuple[str, str, str,
     return (decomposed[0], decomposed[1], git_hash, config_hash)
 
 
+def validate_registration_name(service_instance: str) -> bool:
+    try:
+        service, instance, _, __ = decompose_job_id(service_instance)
+    except InvalidJobNameError:
+        log.error(
+            "Invalid service instance specified. Format is service%sinstance." % SPACER
+        )
+        return False
+    return True
```

Comment:
this was moved but now we're not using it anymore? if we don';t need it for this PR lets revert this change.